incubator/hnc/internal/stats/metrics.go

```diff
@@ -32,6 +32,7 @@ var (
 	objectReconcileConcurrent     = ocstats.Int64("object_reconcile_concurrent_peak", "The peak concurrent object reconciliations happened in the last reporting period", "reconciliations")
 	objectWritesTotal             = ocstats.Int64("object_writes_total", "The number of object writes happened during object reconciliations", "writes")
 	namespaceConditions           = ocstats.Int64("namespace_conditions", "The number of namespaces with conditions", "conditions")
+	objectOverwritesTotal         = ocstats.Int64("object_overwrites_total", "The number of overwritten objects", "writes")
```

Comment:
Maybe use `overwrites`?
objectOverwritesTotal         = ocstats.Int64("object_overwrites_total", "The number of overwritten objects", "**overwrites**")

Reply:
done